.github/workflows/reusable-macos.yml

```
@@ -8,13 +8,14 @@ on:
         required: false
         type: boolean
         default: false
-      os-matrix:
-        required: false
+      os:
+        description: OS to run the job in
```

Just a little nit, both are fine, this a bit shorter:
```suggestion
        description: OS to run the job
```

---

@hugovk do we really need to shorten things here, given that it's not displayed in the UI, except in autocomplete/validation? Or does your variant feel more natural?

---

I think mine is a bit more natural, but if you feel strongly we can keep yours.

---

No, yours is fine. I just wanted to understand better. I was also considering "under" instead of "in".